v2/conn.go

```diff
@@ -733,7 +734,7 @@ func (c *Conn) NewFTEID(ifType uint8, v4, v6 string) (fteidIE *ies.IE) {
 		}
 
 		// Try to mark TEID as taken. Fails if something exists
-		if ok := c.teidSessionMap.tryStore(t, nil); !ok {
+		if ok := c.iteiSessionMap.tryStore(t, nil); !ok {
```

Comment:
we still seem to be allocating for other teids like GTPU when handling GTPC since there is no check?

Reply:
You're right... and finally I understand [what you mentioned in the issue](https://github.com/wmnsk/go-gtp/issues/68#issuecomment-578948136). Since v2.Conn and v1.UPlaneConn could be 1:N, v1.UPlaneConn should manage its TEID by itself to ensure uniqueness. So I updated in the following manner.

* renamed (*Conn).NewFTEID with (*Conn).NewSenderFTEID and removed interface type from parameter, to let users understand it should only be used for its local interface type.
* added (*UPlaneConn).NewFTEID that generates random TEID.